File: tests/test_case4/Main.java
import java.io.FileInputStream;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class Main {
  static int a[] = new int[222];
  public static void main(String[] args) throws FileNotFoundException {
    Scanner scanner = new Scanner(System.in);
    int n = scanner.nextInt();
    for(int i = 0 ; i < n ; ++ i)
      a[i] = scanner.nextInt();
    int sum = 0;
    for(int i = 0 ; i < n ; ++ i)
      sum += a[i];
    System.out.println( sum );
    scanner.close();
  }
}

File: tests/test_case4/main.c
#include <stdio.h>

int n , a[662];

int main() {
  scanf( "%d" , & n );
  for(int i = 1 ; i <= n ; ++ i)
    scanf( "%d" , a + i );
  int sum = 0;
  for(int i = 1 ; i <= n ; ++ i)
    sum += a[i];
  printf( "%d\n" , sum );
  return 0;
}
